flutter/digit-ui-components/digit_components/lib/enum/app_enums.dart

```diff
@@ -65,7 +65,7 @@ enum DropdownType {
 enum InfoType { info, success, error, warning }
 
 /// Enum to define different state for timeline
-enum TimelineStepState { completed, present, future }
+enum TimelineStepState { completed, present, future, failed }
```

Comment:
Change the enum to uppercase

Reply:
enum name is pascal case only and enum value in camel case